### PR TITLE
feat(add-event-volume-scroll) add event on scroll on volume

### PR DESCRIPTION
**Pull request by CYB3RL1F3**
### Context

Need an event triggered when scrolling on volume 

### Changes & Results

- Add new constant EVENTS.VOLUME_SCROLL
- On scroll on canvas, if the slice is in bound, the event EVENTS.VOLUME_SCROLL is triggered with all informations about the current slice

### Testing
- open any volume
- scroll into this one
- listen to the event EVENTS.VOLUME_SCROLL

#### PR

- [X] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [X] My code has been well-documented (function documentation, inline comments,
  etc.)

- [X] I have run the `yarn build:update-api` to update the API documentation, and have
  committed the changes to this PR. (Read more here https://www.cornerstonejs.org/docs/contribute/update-api)

#### Public Documentation Updates

- [X] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [X] "OS: Mac OS 13.6 Ventura 
- [X] "Node version: 20.0.0 
- [X] "Browser:Chrome 119
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: packages/core/src/enums/Events.ts [hunk 1]
```diff
@@ -211,6 +211,11 @@ enum Events {
    */
   VOLUME_SCROLL_OUT_OF_BOUNDS = 'CORNERSTONE_VOLUME_SCROLL_OUT_OF_BOUNDS',
 
+  /**
+   * Triggers when the scroll function is called on a volume.
+   */
+  VOLUME_SCROLL = 'CORNERSTONE_VOLUME_SCROLL',
```
Comment (sedghi) [inline packages/core/src/enums/Events.ts:217]: I noticed that we have STACK_VIEWPROT_SCROLL, so to maintain consistency, could you please think about adding VOLUME_VIEWPORT_SCROLL? I understand that we need to modify VOLUME_SCROLL_OUT_OF_BOUNDS, but that change might cause compatibility issues. I plan to make the update in version 2.x.

**Review by sedghi — COMMENTED** [review]
see my comment thanks

File: packages/core/src/enums/Events.ts (hunk 1 above)
Comment (CYB3RL1F3) [reply packages/core/src/enums/Events.ts:217]: Ok I'll do this!

**Review by sedghi — APPROVED** [review]

**Comment by sedghi** [comment]
Can you please update the api? https://www.cornerstonejs.org/docs/contribute/update-api

**Pull request merged**